src/components/notices-provider/NoticesProvider.jsx

```diff
@@ -22,11 +22,7 @@ const NoticesProvider = ({ children }) => {
     getData();
   }, []);
 
-  return (
-    <div>
-      {children}
-    </div>
```

Comment:
This unnecessary `div` is breaking some global styles with making sure the main content container expands its height to push to the footer to the bottom of the page.